app/main.py

```diff
@@ -1,3 +1,65 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float) -> None:
+        self.km = km
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __add__(self, other: Distance | int) -> Distance:
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        elif isinstance(other, int) or isinstance(other, float):
```

Comment:
```suggestion
        elif isinstance(other, float):
```

Reply:
If I write `elif isinstance(other, float):`, I obtain errors during testing (in cases when int is entered).
In Internet I've found that it is possible to use tuple to check for multiple types with `isinstance`

So, maybe this solution will be better:
`if isinstance(other, (int, float))`

Thanks!

Reply:
`if isinstance(other, (int, float))`
yes it's a solution, GitHub deleted int🧐